peachjam/views/courts.py

```diff
@@ -52,9 +52,16 @@ def get_context_data(self, **kwargs):
             .order_by("judges__name")
             .values_list("judges__name", flat=True)
             .distinct()
+            .exclude(judges=None)
```

Comment:
We're not actually calculating these facets correctly. In a view like https://agp.africanlii.org/legal-instruments/ we calculate the facet values after applying the filters, which helps the user drill down. In this view, we don't, and so applying one facet doesn't impact the others (which is incorrect). Now that we'll have more than one facet, it'll be more obvious.

We should be using the same pattern that is used [here](https://github.com/laws-africa/peachjam/blob/main/peachjam/views/generic_views.py#L58) where we get the base queryset, apply filters (with an exclude), and then count the results.

At the same time, we can optimise that query to return distinct values rather than doing that in python.